code/game/objects/structures/door_assembly_types.dm

```diff
@@ -182,8 +182,8 @@
 
 /obj/structure/door_assembly/door_assembly_cult/unruned/Initialize(mapload)
 	. = ..()
-	icon = SSticker.cultdat?.airlock_unruned_icon_file
-	overlays_file = SSticker.cultdat?.airlock_unruned_overlays_file
+	icon = GET_CULT_DATA(airlock_unruned_icon_file, initial(icon))
+	overlays_file = GET_CULT_DATA(airlock_unruned_overlays_file, initial(overlays_file))
```

Comment:
Initial isn't really needed here, think it's better for readability?

Reply:
Yea